backend/src/main/java/de/bund/digitalservice/ris/search/utils/eli/ManifestationEli.java

```diff
@@ -20,7 +20,11 @@ public record ManifestationEli(
 
   @Override
   public String toString() {
```

Comment:
The manifestation eli in principle should include the extension, therefore, I would suggest that we include in the builder of the manifestation eli the `.xml` and make it the default value if it wasn't set, but then we should pass other extension to the builder of the Manifestation eli since we would be working with more manifestations now for various urls.

Reply:
That is a good point. I was also wondering whether a URI that ends with `.jpg` is a Manifestation ELI, or just something that looks like a Manifestation ELI. In the latter case, I would probably create a new class and revert the changes to `ManifestationEli.java`.